### DISCUSSION TITLE
What is the recommended flow for publishing posts in quarto blog?

### DISCUSSION THREAD

**Discussion by uriahf** [quarto-dev/quarto-cli#5799] [Q&A] [answered]
### Description

Hi all,

What is the recommended flow for publishing blog posts with Quarto?

I thought about opening a branch for each post before publishing it. Do you have any thoughts or resources on the subject?
What if I want to publish some guest posts?

Thanks!

**Comment by mcanouil** [comment, marked as answer]
I personally do everything (or almost) via Pull Request.
And using Netlify to build preview website on my PRs.

There is no such thing as "recommended workflow" or if there is one, it's the one that suits you.

**Reply by uriahf** [reply]
I use netlify as well and I'm really happy with it!
 
Do you think that a branch + pull-request for each post sounds like a reasonable flow?
Do you use freeze and draft parameters?

**Reply by mcanouil** [reply]
> Do you think that a branch + pull-request for each post sounds like a reasonable flow?

All I can say is that it is my workflow.

Because I use the Netlify preview on PR, I don't need draft on top of that.

**Reply by uriahf** [reply]
Great, that's what I had in mind. 
It's a good chance to thank you for your awsome work with awsome quarto! I use it all the time.

That's my blog by the way:
https://rtichoke-blog.netlify.app/
